cmd/create.go

```diff
@@ -26,7 +26,8 @@ import (
 
 // Variables
 var (
-	CreateFiles []string
+	CreateNamespace string
```

Comment:
we can have only one global variable for `namespace` which is action agnostic?

Do we need `CreateNamespace` or `DeleteNamespace` or `ApplyNamespace` ?